### DISCUSSION TITLE
Discord #read-the-description pins are empty

### DISCUSSION THREAD

**Discussion by rhythmique** [Monika-After-Story/MonikaModDev#8527] [QA - tech support + general]
![image](https://user-images.githubusercontent.com/37809643/147705127-95c768c1-6464-413b-a810-a23648358664.png)
Yeah. I think the image explains it.

**Comment by Booplicate** [comment]
It should suggest you to read the rules, did you read them?

**Comment by rhythmique** [comment]
Yeah, I did. I still don't have access to the channels.

**Comment by rhythmique** [comment]
oh wait i'm a dumbass nvm
